data/brands/shop/books.json

```
@@ -11,6 +11,16 @@
     }
   },
   "items": [
+    {
+      "displayName": "QBD Books",
+      "locationSet": {"include": ["au"]},
+      "tags": {
+        "brand": "QBD Books",
+        "brand:wikidata": "Q118994358",
```

Appears to be a duplicate wikidata [Q7270909](https://www.wikidata.org/wiki/Q7270909)
```suggestion
        "brand:wikidata": "Q7270909",
```

---

I didn't create either wikidata pages, but given that Q118994358 has the true brand name of QBD Books and more information than Q7270909, I think we should stick with Q118994358.

---

Oh, I didn't notice that the newer QID was created by another user. Ideal variant - merge these wikidata QIDs. Maybe someone else will do it in the future